betfairlightweight/streaming/cache.py

```diff
@@ -143,15 +143,17 @@ def __init__(self, **kwargs):
         if 'marketDefinition' not in kwargs:
             raise CacheError('"EX_MARKET_DEF" must be requested to use cache')
         self.market_definition = kwargs['marketDefinition']
-        self.runners = [RunnerBook(**i) for i in kwargs.get('rc', [])]
 
+        self.runners = []
         self.runner_dict = {}
         self.market_definition_runner_dict = {}
         self._update_runner_dict()
         self._update_market_definition_runner_dict()
 
+        self.update_cache({'rc': kwargs.get('rc', [])}, self.publish_time)
```

Comment:
Tempted to want to remove this, I try and stay away from any functions being called within an init.

Instead the 'else' can be removed from Stream ln128, thoughts?